apps/avatax/src/modules/avatax/order-confirmed/avatax-order-confirmed-payload-transformer.ts

```diff
@@ -20,6 +26,23 @@ export class AvataxOrderConfirmedPayloadTransformer {
 
     return DocumentType.SalesInvoice;
   }
+  private getSaleorAddress(order: OrderConfirmedSubscriptionFragment) {
+    if (order.shippingAddress) {
+      return ok(order.shippingAddress);
+    }
+
+    if (order.billingAddress) {
+      const message =
+        "OrderConfirmedPayload has no shipping address, falling back to billing address";
+
+      this.logger.warn(message);
+      Sentry.captureMessage(message);
```

Comment:
Why do we put this to sentry? This is not something that should wake up us at night right?
Adding a logger message is good enough.

If we don't have shipping AND billing address, that means contract is broken. This goes to sentry

Reply:
You're right